linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -627,6 +628,34 @@ impl LocalNet {
         self.running_validators.insert(validator, validator_proxy);
         Ok(())
     }
+
+    /// Terminates all the processes of a given `validator`.
+    pub async fn stop_validator(&mut self, validator: usize) -> Result<()> {
+        if let Some(mut validator) = self.running_validators.remove(&validator) {
+            validator.terminate().await?;
+        }
```

Comment:
I would prefer an error to be reported here.

Reply:
`tracing::warn!`

Reply:
Done.